jbpm-task-assigning/kie-server-client-task-assigning/src/test/java/org/kie/server/client/IntegrationTests.java

```diff
@@ -0,0 +1,247 @@
+    }
+
+    private void testFindTasksWithTaskAssigningClientTimes(int pageSize, TaskInputVariablesReadMode inputVariablesReadMode) {
+        List<String> status = Arrays.asList(Ready,
+                                            Reserved,
+                                            InProgress,
+                                            Suspended);
+
+        registerStartTime("findTasks");
+        List<TaskData> list = client.findTasks(0L, status, null, 0, pageSize, inputVariablesReadMode).getItems();
+        StopWatch stopWatch = registerEndTime("findTasks");
+        System.out.println("Tasks size with variablesReadMode: " + inputVariablesReadMode + " pageSize: " + pageSize + " totalTasks: " + list.size() + " in " + printTime((stopWatch)));
+    }
+
+    @Test
+    public void findTasksWithTasksReaderAllVariablesReadTime() {
+        pageSizes.forEach(pageSize -> findTasksWithTasksReaderTimes(pageSize, TaskInputVariablesReadMode.READ_FOR_ALL));
+    }
+
+    @Test
+    public void findTasksWithTasksReaderVariablesWhenPlanningTaskIsNull() {
+        pageSizes.forEach(pageSize -> findTasksWithTasksReaderTimes(pageSize, TaskInputVariablesReadMode.READ_WHEN_PLANNING_TASK_IS_NULL));
+    }
+
+    @Test
+    public void findTasksWithTasksReaderNoVariables() {
+        pageSizes.forEach(pageSize -> findTasksWithTasksReaderTimes(pageSize, TaskInputVariablesReadMode.DONT_READ));
+    }
+
+    private void findTasksWithTasksReaderTimes(int pageSize, TaskInputVariablesReadMode inputVariablesReadMode) {
+        List<String> status = Arrays.asList(Ready,
+                                            Reserved,
+                                            InProgress,
+                                            Suspended);
+
+        registerStartTime("findTasks");
+        List<TaskData> list = TaskDataReader.from(client).readTasks(0L, status, null, pageSize, inputVariablesReadMode).getTasks();
+        StopWatch stopWatch = registerEndTime("findTasks");
+        System.out.println("With TaskDataReader, Tasks size with variablesReadMode: " + inputVariablesReadMode + " pageSize: " + pageSize + " totalTasks: " + list.size() + " in " + printTime(stopWatch));
+    }
+
+    private static String printTime(StopWatch stopWatch) {
+        return stopWatch.toString();
+    }
+
+    @Test
+    public void createProcessInstances() {
+
+        int processInstancesSize = 5;
+        List<Long> processInstances = new ArrayList<>();
+
+        ProcessServicesClient processServices = newKieServicesClient().getServicesClient(ProcessServicesClient.class);
+
+        long processInstanceId;
+        for (int i = 0; i < processInstancesSize; i++) {
+            HashMap inputParams = new HashMap();
+            inputParams.put("processVar1", "generatedValue.for.taskInput1_" + i);
+            inputParams.put("processVar2", "generatedValue.for.taskInput2_" + i);
+            processInstanceId = processServices.startProcess(CONTAINER_ID, PROCESS_ID);
```

Comment:
`inputParams` is unused. Maybe it should be passed here?

Btw, how do I run `IntegrationTests`? If I run it from IDEA I get this:

> org.kie.server.common.rest.NoEndpointFoundException: No available endpoints found

If I do `mvn clean install` the test doesn't seem to be run at all.

Reply:
The same for me.

Reply:
same here, this class will be removed.